CMakeLists.txt

```
@@ -358,6 +355,20 @@ else()
       POSITION_INDEPENDENT_CODE true)
 endif()
 
+
+# Now target jansson is declared, set per-target values
+
+target_compile_definitions(jansson PUBLIC JANSSON_USING_CMAKE)
+target_compile_definitions(jansson PRIVATE HAVE_CONFIG_H)
```

```suggestion
target_compile_definitions(jansson PUBLIC JANSSON_USING_CMAKE PRIVATE HAVE_CONFIG_H)
```

Might be good to split across multiple lines for clarity, but there's no need to use two separate invocations of the `target_compile_definitions` command here.